doc/visualizations.rst

```diff
@@ -8,11 +8,20 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and input data (`X`, `y`).
+The `from_predictions` method creates a `Display` object from true and predicted values (`y_test`, `y_pred`), which
+is useful when you only want to compute the predictions once.
+
+The `Display` object stores the computed values required for plotting with Matplotlib.
+These values can either be passed directly via `from_predictions`, or derived from an estimator
+and sample data using `from_estimator`.
```

Comment:
Uhm I'm a bit confused here. In the previous sentence, it looks like the "computed values" are related to the predictions rather then the derivative values from those predictions (e.g. metrics or feature importance).

So we pass the raw predictions to `from_predictions` or get them from the estimator via `from_estimator` and then compute some derivatives and those values are stored in the display. By doing so, we avoid to recompute them when we call the `plot` method.